Source/Core/Core/HW/MemmapFunctions.cpp

```diff
@@ -965,7 +873,8 @@ static u32 TranslateBlockAddress(const u32 addr, const XCheckTLBFlag _Flag)
 }
 
 // Translate effective address using BAT or PAT.  Returns 0 if the address cannot be translated.
-u32 TranslateAddress(const u32 _Address, const XCheckTLBFlag _Flag)
+template <const XCheckTLBFlag _Flag>
```

Comment:
Then you could just make it static... but it's also used from PPCDebugInterface.cpp.